libs/portals/admin/ids-admin/src/components/Applications/Applications.graphql

```diff
@@ -0,0 +1,19 @@
+# Get Applications by ID
+query GetApplicationsById($input: ApplicationsInput!) {
+  authAdminApplications(input: $input) {
+    data {
+      applicationId
```

Comment:
Will this be reused in `GetApplication` query? Just thinking if the content of data should be a fragment

Reply:
It will not be reused, to GetApplication we will need another input model